docs/source/examples/examples.myst

```diff
@@ -8,6 +8,7 @@ ibmq-backends.rst
 hamiltonians.rst
 simple_landscape.rst
 maxcut-demo.myst
+vqe-demo.myst
```

Comment:
I think this should be pyquil_demo.myst now.

Reply:
Done!